template/function/serverless.yml

```
@@ -1,21 +1,29 @@
-service: app
+service: TPL_SERVICE_NAME
 
 provider:
-    name: aws
-    region: us-east-1
-    runtime: provided.al2
+  # general settings
+  name: aws
+  stage: dev
+  region: TPL_REGION
+  profile: default
+  deploymentMethod: direct
```

```suggestion
  # Use the newer and faster deployment method
  deploymentMethod: direct
```

As a user I hate when there's boilerplate I don't understand. If we can't remove the boilerplate, we can explain it.